extensions/amp-analytics/0.1/vendors/upscore.json

```diff
@@ -2,10 +2,11 @@
   "requests": {
     "host": "https://hit-pool.upscore.com/amp?",
     "basePrefix": "u_id=${clientId(upscore)}&hit_id=${pageViewId}&scTop=${scrollTop}&scHeight=${scrollHeight}&vHeight=${viewportHeight}&domain=${domain}&load=${domInteractiveTime}&timespent=${totalEngagedTime}",
-    "initialHit": "author=${author}&creator=${creator}&o_id=${object_id}&o_type=${object_type}&pubdate=${pubdate}&ref=${documentReferrer}&section=${section}&url=${canonicalUrl}&agent=${userAgent}&location=${ampGeo(ISOCountry)}&c_keys=${customKeys}&c_values=${customValues}",
-    "finalbeat": "${host}${basePrefix}&type=final",
+    "initialHit": "author=${author}&o_id=${object_id}&o_type=${object_type}&pubdate=${pubdate}&ref=${documentReferrer}&section=${section}&taxonomy=${taxonomy}&url=${canonicalUrl}&agent=${userAgent}&location=${ampGeo(ISOCountry)}&c_keys=${customKeys}&c_values=${customValues}&content_type=${content_type}&user_status=${user_status}&content_blocked=${content_blocked}",
+    "finalBeat": "${host}${basePrefix}&type=final",
```

Comment:
I reverted to the old request names. My understanding was that it should work fine since only we are defining the triggered request name and the trigger in the same configuration. The users are never calling the request by name. Did I get that wrong?